### ISSUE TITLE
Training area

### ISSUE THREAD

**Issue by HobnobberJunior** [SmashMC-Development/Suggestions#838]
Would be nice to have the blissey spawns back at the training area lvl 60-80, so you do not just stand and level up mons for ages. Maybe even have more blisseys to spawn than before since there isn't a reason to just stand mindlessly clicking buttons just to level up mons.

**Comment by ghost** [comment]
I think it is suppose to take time so you don't get wz ready pokemon easily. But, I could be wrong.

**Comment by HobnobberJunior** [comment]
I do not quite agree with this. You can just buy lvl 100s for 5-10k, which you can make in no time. The only time you would really use the training area is yo level op bred mons which sucks if u already have a bunch of lvl 100s.

**Comment by ghost** [comment]
I think it would be very using to get lvl 100 BPs if they make the change you are suggesting which would make ppl care less about what they lose in wz. Right now the only incentive to steal BP in wz is because it's hard to level up...at least for me. But like I said I could be wrong.

**Comment by Chowlien** [comment]
> I think it would be very using to get lvl 100 BPs if they make the change you are suggesting which would make ppl care less about what they lose in wz. Right now the only incentive to steal BP in wz is because it's hard to level up...at least for me. But like I said I could be wrong.

I agree with this point but if players aren't levelling their bps because it takes too long, it also makes wz very sad to play in. There are hardly any players with any bps/actual teams. If it were easier to train pokemon to level 100 there would be more players in wz with actual teams. Not only this but wz have lvl 90-100 pokemon spawn very often meaning there's not much point in bringing bps if it's easier just to bring wz poke in wz. What I'm trying to say is the 'competitive' aspect of wz is lost now cause it's too hard to train bps, it also means players are buying less bps for this very same reason too.

**Comment by Mikirae** [comment]
It was initially done this way as a sort of way to incentivize going to boss towers more and having people grind more, but it seems it might be doing more harm than good. We'll look into this and see if we can come up with a solution that makes everyone happy :)

**Comment by ghost** [comment]
> It was initially done this way as a sort of way to incentivize going to boss towers more and having people grind more, but it seems it might be doing more harm than good. We'll look into this and see if we can come up with a solution that makes everyone happy :)

I think ppl would be incentivized to go to boss tower if you removed fossils and berries from the drop tables. Probably gems too. No one uses gems.

**Comment by Mikirae** [comment]
Put it on a suggestion for me and I'll take a look at it with everyone <3
Wouldn't mind removing certain things really

**Comment by ghost** [comment]
> > I think it would be very using to get lvl 100 BPs if they make the change you are suggesting which would make ppl care less about what they lose in wz. Right now the only incentive to steal BP in wz is because it's hard to level up...at least for me. But like I said I could be wrong.
> 
> I agree with this point but if players aren't levelling their bps because it takes too long, it also makes wz very sad to play in. There are hardly any players with any bps/actual teams. If it were easier to train pokemon to level 100 there would be more players in wz with actual teams. Not only this but wz have lvl 90-100 pokemon spawn very often meaning there's not much point in bringing bps if it's easier just to bring wz poke in wz. What I'm trying to say is the 'competitive' aspect of wz is lost now cause it's too hard to train bps, it also means players are buying less bps for this very same reason too.

Players who don't really want to battle will always bring throw teams irrespective of the ease of training. If you ease the training , they will just train throw teams faster XD. Saying that I do believe that people who are new to competitive battling and would like to experiment with their teams would benefit from faster training...but I don't think it would be worth changing it just for them. I don't think it effects competitiveness at all...I personally don't like to lose in wz....so I have a BP team always. I believe same is true for other competitive players on the server atleast on Groudon. The fact is some ppl don't like to battle and they will never bring a real team to wz....and that's fine.

**Comment by HobnobberJunior** [comment]
I had to level up a bunch of bps yesterday and it took ages. It wasn't fun at all and if not because ive done a bunch of raids, I would still not be done. I do not think making the breeding area better would make people bring more throws to wz. People who wants to bring throws can just buy them, they have no real reason to use the grinding area.
